src/backend/base/langflow/base/models/aws_constants.py

```diff
@@ -0,0 +1,70 @@
+AWS_MODEL_IDs = [
+    "amazon.titan-text-express-v1",
+    "amazon.titan-text-lite-v1",
+    "amazon.titan-text-premier-v1:0",
+    "amazon.titan-embed-text-v1",
+    "amazon.titan-embed-text-v2:0",
+    "amazon.titan-embed-image-v1",
+    "amazon.titan-image-generator-v1",
+    "anthropic.claude-v2",
+    "anthropic.claude-v2:1",
+    "anthropic.claude-3-sonnet-20240229-v1:0",
+    "anthropic.claude-3-haiku-20240307-v1:0",
+    "anthropic.claude-3-opus-20240229-v1:0",
+    "anthropic.claude-instant-v1",
+    "ai21.j2-mid-v1",
+    "ai21.j2-ultra-v1",
+    "cohere.command-text-v14",
+    "cohere.command-light-text-v14",
+    "cohere.command-r-v1:0",
+    "cohere.command-r-plus-v1:0",
+    "cohere.embed-english-v3",
+    "cohere.embed-multilingual-v3",
+    "meta.llama2-13b-chat-v1",
+    "meta.llama2-70b-chat-v1",
+    "meta.llama3-8b-instruct-v1:0",
+    "meta.llama3-70b-instruct-v1:0",
+    "mistral.mistral-7b-instruct-v0:2",
+    "mistral.mixtral-8x7b-instruct-v0:1",
+    "mistral.mistral-large-2402-v1:0",
+    "mistral.mistral-small-2402-v1:0",
+    "stability.stable-diffusion-xl-v0",
+    "stability.stable-diffusion-xl-v1",
+]
```

Comment:
We probably shouldn't put image generation and embeddings models in this list. I've removed some but there may be others
```suggestion
AWS_MODEL_IDs = [
    "amazon.titan-text-express-v1",
    "amazon.titan-text-lite-v1",
    "amazon.titan-text-premier-v1:0",
    "anthropic.claude-v2",
    "anthropic.claude-v2:1",
    "anthropic.claude-3-sonnet-20240229-v1:0",
    "anthropic.claude-3-haiku-20240307-v1:0",
    "anthropic.claude-3-opus-20240229-v1:0",
    "anthropic.claude-instant-v1",
    "ai21.j2-mid-v1",
    "ai21.j2-ultra-v1",
    "cohere.command-text-v14",
    "cohere.command-light-text-v14",
    "cohere.command-r-v1:0",
    "cohere.command-r-plus-v1:0",
    "meta.llama2-13b-chat-v1",
    "meta.llama2-70b-chat-v1",
    "meta.llama3-8b-instruct-v1:0",
    "meta.llama3-70b-instruct-v1:0",
    "mistral.mistral-7b-instruct-v0:2",
    "mistral.mixtral-8x7b-instruct-v0:1",
    "mistral.mistral-large-2402-v1:0",
    "mistral.mistral-small-2402-v1:0",
]
```

Reply:
verified that the updated list is not image or embeddings model.